src/ImpactX.cpp

```diff
@@ -117,4 +117,18 @@ namespace impactx
         } // end step loop
     }
 
+    void ImpactX::initElements ()
+    {
+        // initialize element sequence
+        myelements = { };
```

Comment:
The list will be automatically constructed in `ImpactX::ImpactX` as an empty container.
Here, it's good to make sure the list is empty, e.g., if we want to re-initialize it.

`= {}` is semantically equivalent to `.clear()`, but the latter might be a bit more explicit:
```suggestion
        // make sure the element sequence is empty
        myelements.clear();
```